temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message EndpointSpec {
+    // The name of the endpoint. Must be unique within an account.
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    // This field is mutable.
+    string name = 1;
+
+    // Indicates where the endpoint should forward received nexus requests to. 
+    EndpointTargetSpec endpoint_target_spec = 2;
+
+    // The set of policies (e.g. authorization) for the endpoint. Each request's caller
+    // must match with at least one of the specs to be accepted by the endpoint.
+    // This field is mutable.
+    repeated EndpointPolicySpec policy_specs = 3;
+
+    // Optional description. Can be in markdown for rendering in the UI.
+    // Note that this is an experimental field that may be deprecated in the future
+    string description = 4;
+}
+
+message EndpointTargetSpec {
+    // The type of target that this endpoint routes Nexus requests to.
+    // Possible values are: cloud_task_queue
+    string target_type = 1;
+
+    // A target spec for routing nexus requests to a specific cloud namespace worker.
+    CloudWorkerTargetSpec cloud_worker_target_spec = 2;
+}
+
+message CloudWorkerTargetSpec {
+    // The target cloud namespace to route requests to. Namespace must be in same account as the endpoint. This field is mutable.
+    string namespace = 1;
+
+    // The task queue on the cloud namespace to route requests to. This field is mutable.
+    string task_queue = 2;
+}
+
+message EndpointPolicySpec {
+    // The policy set on this endpoint authorization spec.
+    // Possible values are: allowed_cloud_namespace
+    string policy_type = 1;
```

Comment:
Is this necessary? Can the presence of the next field be enough to tell you what the policy type is?

Reply:
removed